gobblin-data-management/src/test/java/gobblin/data/management/copy/CopyableFileTest.java

```diff
@@ -33,7 +33,7 @@ public void testSerializeDeserialze() throws Exception {
         new CopyableFile(new FileStatus(10, false, 12, 100, 12345, new Path("/path")), new Path("/destination"),
             new Path("/relative"), new OwnerAndPermission("owner", "group", FsPermission.getDefault()),
             Lists.newArrayList(new OwnerAndPermission("owner2", "group2", FsPermission.getDefault())),
-            "checksum".getBytes(), PreserveAttributes.fromMnemonicString(""), "");
+            "checksum".getBytes(), PreserveAttributes.fromMnemonicString(""), "", 0, 0);
```

Comment:
This signature is wrong isn't it? Last two arguments shouldn't be there. Similar for other test changes.